packages/onchain-claims/contracts/Migrations.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.4.22 <0.9.0;
+
+contract Migrations {
```

Comment:
Just curious why we need this? I though that migrations are used to track deployment, but here we aren't deploying anything

Reply:
Yes, we are not deploying anything as of now. But for testing different use cases ( like transferring of ownership, we need to deploy again for testing ). So I though it would be good to keep it for future testing as well.